fms/modules/feedforward.py

```diff
@@ -6,7 +6,7 @@
 import torch.nn.functional as F
 from torch.distributed.distributed_c10d import ProcessGroup
 
-import fms.triton.moe_kernel as moe_kernel  # registers the PT custom ops
+import fms.triton.pytorch_ops as triton_ops  # registers the PT custom ops
```

Comment:
is there some way we can add a test (e.g. that some basic imports correctly import without triton installed)?

Reply:
I think the main thing I see is if I don't have triton installed I can't import fms.models, so just a test that made that import in an env without triton would cover most potential failure cases I think. not just for this particular fix, but as we add other triton kernels in the future too